docs/user/aws/install_upi.md

```diff
@@ -0,0 +1,432 @@
+    metadata:
+      creationTimestamp: null
+      labels:
+        machine.openshift.io/cluster-api-cluster: cuppett-tkh7l
+        machine.openshift.io/cluster-api-machine-role: worker
+        machine.openshift.io/cluster-api-machine-type: worker
+        machine.openshift.io/cluster-api-machineset: cuppett-tkh7l-worker-us-east-2a
+    spec:
+      metadata:
+        creationTimestamp: null
+      providerSpec:
+        value:
+          ami:
+            id: ami-0eecbb884c8b35b1e
+          apiVersion: awsproviderconfig.openshift.io/v1beta1
+          blockDevices:
+          - ebs:
+              iops: 0
+              volumeSize: 120
+              volumeType: gp2
+          credentialsSecret:
+            name: aws-cloud-credentials
+          deviceIndex: 0
+          iamInstanceProfile:
+            id: cuppett-tkh7l-worker-profile
+          instanceType: m4.large
+          kind: AWSMachineProviderConfig
+          metadata:
+            creationTimestamp: null
+          placement:
+            availabilityZone: us-east-2a
+            region: us-east-2
+          publicIp: null
+          securityGroups:
+          - filters:
+            - name: tag:Name
+              values:
+              - cuppett-tkh7l-worker-sg
+          subnet:
+            filters:
+            - name: tag:Name
+              values:
+              - cuppett-tkh7l-private-us-east-2a
+          tags:
+          - name: kubernetes.io/cluster/cuppett-tkh7l
+            value: owned
+          userDataSecret:
+            name: worker-user-data
+      versions:
+        kubelet: ""
+status:
+  fullyLabeledReplicas: 1
+  observedGeneration: 1
+  replicas: 1
+```
+
+At this point, you'd edit the YAML to update the relevant values to match your UPI installation.
+
+```console
+$ oc edit machineset --namespace openshift-machine-api cuppett-tkh7l-worker-us-east-2a
+machineset.machine.openshift.io/cuppett-tkh7l-worker-us-east-2a edited
+```
+
+Once the Machine API has a chance to reconcile and begin launching hosts with the correct attributes, you 
+should start to see new output in your EC2 console and oc commands.
+
+```console
+$ oc get machines --namespace openshift-machine-api
+NAME                                    INSTANCE              STATE     TYPE         REGION      ZONE         AGE
+cuppett-tkh7l-worker-us-east-2a-hxlqn   i-0e7f3a52b2919471e   pending   m4.4xlarge   us-east-2   us-east-2a   3s
+```
+
+### Option 2: Manually Launching Worker Instances
+
+The recommended worker launch and other necessary DNS entries for etcd are provided within a CloudFormation
+template [here](../../../upi/aws/cloudformation/06_cluster_worker_node.yaml). You can launch a CloudFormation stack
+to manage each individual worker. A similar launch configuration could be used by outside automation or AWS auto
+scaling groups. 
+
+## Configure Router for UPI DNS
+
+TODO: Identify changes needed to Router or Ingress for DNS *.apps registration or LoadBalancer creation.
```

Comment:
nit: backticks: `*.apps`, but...

Are we comfortable landing this with a TODO?  I'm fine addressing `*.apps` routing or not, but if we don't address it, I'd rather have a sentence or two about why we aren't and what we expect as the consequences instead of just saying "TODO".  But I'm not clear enough on this yet to float potential wording.  Do we need an `*.apps` entry in the private zone or just the public zone?  Will the ingress operator attempt to create that route (or those routes) if they don't exist with this UPI workflow?  If we don't have time to dig into this now, I'd rather just scrap this subsection and add a note to the commit message with "We didn't have time to dig into ingress DNS for `*.apps`" or some such.

Reply:
> nit: backticks: *.apps, but...

Added backticks with dafdaa034 -> bdae4f939.  I've punted on the bigger picture, but we can circle back once we have CI for this workflow.